api/google-manager/services/google-manager.js

```diff
@@ -96,6 +96,23 @@ class GoogleManager {
     }
   }
 
+  async getGroup(groupEmail) {
+    try {
+      const admin = await google.admin({
+        version: 'directory_v1',
+        auth: this.adminAuth
+      });
+
+      const group = await admin.groups.get({
+        groupKey: groupEmail
+      });
+
+      return group.data;
+    } catch(err) {
+      console.error(`Google Admin Directory API returned error ${err} when getting group`);
```

Comment:
Does `console.error` return an error to the caller?

Reply:
I don't think so. I should probably return an error.